src/controller/python/chip/ChipStack.py

```diff
@@ -255,11 +255,7 @@ def CallAsyncWithCompleteCallback(self, callFunct):
         if not res.is_success:
             self.completeEvent.set()
             raise res.to_exception()
-        while not self.completeEvent.isSet():
-            if self.blockingCB:
```

Comment:
How about removing this `blockingCB` completely? This variable and the setter.

Reply:
Right, of course, something went wrong here while gitting. Now it should be fine.